pyuploadcare/dj/models.py

```diff
@@ -52,6 +54,16 @@ def formfield(self, **kwargs):
 
         return models.Field.formfield(self, **kwargs)
 
+    def validate(self, value, model_instance):
+        super(FileField, self).validate(value, model_instance)
+
+        try:
+            value.info()
+        except InvalidRequestError:
+            raise ValidationError(_('UUID to store could not be found on server: %(uuid)s'),
```

Comment:
Yeah, that's better! I changed it to `'The file with UUID %(uuid)s could not be found in your Uploadcare project'`.